py/shared/donorfy.py

```diff
@@ -137,13 +138,16 @@ class DonorfyActor(BaseActor):
                 'price: {price_text}\n'
                 'description: {long_description}'
             ).format(**evt),
-            Code1=evt['link'],
-            Code2=evt['location_name'],
-            Code3=evt['short_description'],
             Number1=evt['ticket_limit'],
             Date1=format_dt(created),
             Date2=format_dt(start_ts),
         )
+
+        codes = evt['link'], evt['location_name'], evt['short_description']
+        for i, v in enumerate(codes, start=1):
+            if v:
+                data[f'Code{i}'] = shorten(v, width=50, placeholder='...')
```

Comment:
answering #184: No, I haven't changed `Notes`, just altered codes to:
1. only be included if a value exists for them
2. be truncated to 50 characters max.